lib/modules/keyboardNav.js

```diff
@@ -837,194 +869,56 @@ module.followSubreddit = function(newWindow) {
 	}
 };
 
-module.moveUp = function() {
-	const selected = SelectedEntry.selectedThing();
-	if (!selected) return;
+function showNextMedia(oldThing, newThing, direction) {
+	if (
+		oldThing && newThing &&
+		module.options.mediaBrowseMode.value &&
+		isPageType('linklist', 'modqueue', 'search')
+	) {
+		return ShowImages.mediaBrowseMode(oldThing, newThing, direction);
+	}
+}
 
-	_moveUp(selected.thing);
-};
+function move(target, { scrollStyle = module.options.scrollStyle.value, tryMediaBrowse = false } = {}) {
+	const selected = SelectedEntry.selectedThing();
 
-module.moveUpComment = function() {
-	module.moveUp();
-};
+	if (target instanceof Function) {
```

Comment:
Nit: `typeof target === 'function'` is usually used to do this (and is technically more accurate, though that doesn't really matter here)